lblod/frontend-loket
# [DGS-161] Add a `subsidies-external` feature flag

## ID
DGS-161

## Description

Add a `subsidies-external` feature flag to quickly change from internal subsidiemodule in loket to external subsidies application redirection.

In the docker-compose.override.yml for production deployment should the following environment variables be added with the final values (TBD):
```
EMBER_SUBSIDIES_URL: "https://subsidiepunt.lokaalbestuur.vlaanderen.be/"
EMBER_FEATURE_SUBSIDIES_EXTERNAL: "false"
```

## Type of change


## How to test

During development, update the `config/environment.js` environment variables of `subsidies-external` to true/false and `subsidiesUrl` to a new url. The subsidy module should redirect to the url when the feature flag is set to true.
